_scalingo_api/log_drains.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Log drains
```

Comment:
```suggestion
title: Log Drains
```

Title in English have upper case for (almost) all words: https://www.notion.so/scalingooriginal/Writing-Documentation-at-Scalingo-bb7985d6ea16455894417e36ede09cea#3ff5d385e97e45bda37bc80cd618c725